.travis.yml

```diff
@@ -142,27 +151,10 @@ jobs:
 
         " && false; }
 
-    - name: "analyze hadoop 3 dependencies"
-      script: |-
-        MAVEN_OPTS='-Xmx3000m' ${MVN} ${MAVEN_SKIP} dependency:analyze -DoutputXML=true -DignoreNonCompile=true -DfailOnWarning=true -Phadoop3 || { echo "
-
-        The dependency analysis has found a dependency that is either:
-
-        1) Used and undeclared: These are available as a transitive dependency but should be explicitly
-        added to the POM to ensure the dependency version. The XML to add the dependencies to the POM is
-        shown above.
-
-        2) Unused and declared: These are not needed and removing them from the POM will speed up the build
-        and reduce the artifact size. The dependencies to remove are shown above.
-
-        If there are false positive dependency analysis warnings, they can be suppressed:
-        https://maven.apache.org/plugins/maven-dependency-plugin/analyze-mojo.html#usedDependencies
-        https://maven.apache.org/plugins/maven-dependency-plugin/examples/exclude-dependencies-from-dependency-analysis.html
-
-        For more information, refer to:
-        https://maven.apache.org/plugins/maven-dependency-plugin/analyze-mojo.html
-
-        " && false; }
+    - <<: *analyze_dependencies
+      name: "analyze hadoop 3 dependencies"
```

Comment:
Can we introduce a dummy code change and role it back to see if `analyze hadoop 3 dependencies` works. 
We can roll it back prior to merge.

Reply:
https://app.travis-ci.com/github/apache/druid/jobs/587017381 - does this work?

Reply:
I can also add analyze dependencies, analyze hadoop 3 dependencies to check_test_suite.py and then revert it back.